### ISSUE TITLE
Add the `getEnv` function

### ISSUE THREAD

**Issue by edouardmisset** [edouardmisset/my-learning-curve#9]
For Vite :

```ts
const libraryPrefix = 'VITE_'

export function getEnv(variable: string): string {
  const value =
    import.meta.env[variable] || import.meta.env[`${libraryPrefix}${variable}`]
  if (value === undefined) {
    throw new TypeError(`It seems like the variable "${variable}" is not set in the environment (\`.env\` file).
    @Dev: Did you forget to execute "cp .env.dev .env" and adjust variables in the .env file to match your own environment ?`)
  }
  return value
}
```

**Comment by edouardmisset** [comment]
Also add usual variables :

```ts
export const inProdEnv = import.meta.env.PROD
export const inDevEnv = import.meta.env.DEV
export const inTestEnv = import.meta.env.MODE === 'test'

export const PUBLIC_URL = import.meta.env.BASE_URL
```

**Comment by edouardmisset** [comment]
filename : `// env.ts`

**Comment by edouardmisset** [comment]
Closed by c44e446
